Makefile

```diff
@@ -586,13 +591,16 @@ ${OUT}/istio-sidecar.deb: ${ISTIO_BIN}/envoy ${ISTIO_BIN}/pilot-agent ${ISTIO_BI
 	   --config-files /var/lib/istio/envoy/envoy.json \
 	   --description "Istio" \
 	   src/istio.io/istio/tools/deb/istio-start.sh=/usr/local/bin/istio-start.sh \
-	   src/istio.io/istio/tools/deb/istio-iptables.sh=/usr/local/bin/istio-iptables/sh \
+	   src/istio.io/istio/tools/deb/istio-iptables.sh=/usr/local/bin/istio-iptables.sh \
 	   src/istio.io/istio/tools/deb/istio.service=/lib/systemd/system/istio.service \
-	   src/istio.io/istio/security/tools/deb/istio-auth-node-agent.service=/lib/systemd/system/istio-auth-node-agent.service \
+	   src/istio.io/istio/tools/deb/istio-auth-node-agent.service=/lib/systemd/system/istio-auth-node-agent.service \
 	   bin/envoy=/usr/local/bin/envoy \
 	   bin/pilot-agent=/usr/local/bin/pilot-agent \
-	   bin/node_agent=/usr/local/istio/bin/node_agent \
-	   src/istio.io/istio/tools/deb/sidecar.env=/var/lib/istio/envoy/sidecar.env \
+	   bin/node_agent=/usr/local/bin/node_agent \
+	   bin/istioctl=/usr/local/bin/istioctl \
+	   bin/mixs=/usr/local/bin/mixs \
+	   bin/pilot-discovery=/usr/local/bin/pilot-discovery \
+           src/istio.io/istio/tools/deb/sidecar.env=/var/lib/istio/envoy/sidecar.env \
```

Comment:
Looks like there's an extraneous space char before the "src"

Reply:
Fixed.